### PR TITLE
Disable tracking improvements

### PR DESCRIPTION
**Pull request by ecwillis**

### CODE REVIEW

File: routes/view.js [hunk 1]
```diff
@@ -88,11 +88,12 @@ exports.testView = {
       payload(server, request, done) {
         const email = request.params.email;
         const testPath = `${server.settings.app.views.path}/${email}/test.json`;
+        const disableTracking = (request.query.disableTracking !== 'false');
```
Comment (jgallen23) [inline routes/view.js]: seems confusing to have enableMetrics and disableTracking

File: test/method.getEmailDetails.test.js [hunk 2]
```diff
@@ -46,7 +46,8 @@ tap.test('getEmailDetails - with yaml', (assert) => {
       data: {
         firstName: 'bob',
         lastName: 'smith',
-        serviceName: 'test city'
+        serviceName: 'test city',
+        trackingPixel: ''
```
Comment (jgallen23) [inline test/method.getEmailDetails.test.js]: why was this added?

File: routes/view.js (hunk 1 above)
Comment (ecwillis) [inline routes/view.js]: It kind of is. However - enableMetrics is global and can't be easily over written ... and this would allow disabling the tracking on a specific view.

File: test/method.getEmailDetails.test.js (hunk 2 above)
Comment (ecwillis) [inline test/method.getEmailDetails.test.js]: I added a default for the tracking pixel of a '' so that if it is used in a template it is not trying to use an undefined variable. So now it is an empty string.

Comment (jgallen23) [inline test/method.getEmailDetails.test.js]: got it, so we don't have to include this all the time, right?
